pkg/features/feature_flags.go

```diff
@@ -22,6 +22,11 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
+// Feature flag constant strings
```

Comment:
#2323 adds this constant (though in a different package, https://github.com/vmware-tanzu/velero/pull/2323/files#diff-193655ac4b146a95ba1bbfd621ad7132R41) so you'll want to use that one once it merges

Reply:
💯 I wanted to get this change tested out and did what was best to avoid being blocked on #2323